client/interface.go

```diff
@@ -15,8 +15,9 @@ import (
 // APIClient is an interface that clients that talk with a docker server must implement.
 type APIClient interface {
 	ClientVersion() string
-	ContainerAttach(ctx context.Context, container string, options types.ContainerAttachOptions) (types.HijackedResponse, error)
```

Comment:
Why `ContainerAttach` and `ContainerCommit` disappear ? (typos :stuck_out_tongue_closed_eyes:)